### PR TITLE
Started documentation for uLanding radar

### PR DESCRIPTION
**Pull request by tumbili**

### CODE REVIEW

File: book/uart-ulanding-radar.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Enable the driver for your hardware
+Currently, this radar device is supported by any hardware which runs the OS NuttX and which can offer a serial port for the
+interface. Since flash space is small on some hardware you may have to enable building the driver for your target yourself.
+To do so add the following line to the cmake config file which correspond to the target you want to build for:
+```
+drivers/ulanding
+```
+
+So let's say you want to use the radar on a  Pixhawk, then you would add the upper line to the file
```
Comment (LorenzMeier) [inline book/uart-ulanding-radar.md]: This is a bit colloquial 8)

Comment (tumbili) [inline book/uart-ulanding-radar.md]: @LorenzMeier Hehe, you know I want the reader to feel like his my friend and I take care of him ;-)

**Comment by tumbili** [comment]
@LorenzMeier Updated. Anything else we should add? I can offer a picture with the hardware connected.

**Comment by LorenzMeier** [comment]
The picture would be helpful I guess.

File: book/uart-ulanding-radar.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+You will have to tell the sytem to start the driver for the radar during sytem startup.
+You can simply add the following line to an [extras.txt](advanced-system-startup.md) file located on your SD card.
+```
+ulanding_radar start /dev/serial_port
+```
+
+In the upper command you will have to replace the last argument with the serial port you have connected the hardware to.
+If you don't specify any port the driver will use /dev/ttyS2 which is the TELEM2 port on Pixhawk.
+
+<aside class="notice">
```
Comment (LorenzMeier) [inline book/uart-ulanding-radar.md]: This is not any more supported.

**Pull request merged**